docs/LINUX.md

```diff
@@ -1,125 +1,141 @@
 # LogDNA Agent on Linux distributions
 
-The agent supports 64-bit (`x86_64`) Linux distributions and provides rpm and deb packages.
+The LogDNA Agent for Linux collects log data from your Linux environment. The agent is available as a Linux package for Debian-based distributions using `apt` and Red Hat-based distributions using `yum`. If you use a different package manager that does not support `.deb` or `.rpm` files, you can still use the agent by [manually compiling the binary](https://github.com/logdna/logdna-agent-v2/blob/eb06d4f3f7c1033b494f1f0439957f96533f9225/docs/README.md#building-agent-binary-on-linux).
 
-## Installing
+## Table of Contents
```

Comment:
To simplify reading for new users, I think the order should be:

1. Installing
2. Usage
3. Upgrading from legacy Linux Agent

That way, new users can read the required steps in order (1 & 2) without jumping through the page.

Reply:
@jorgebay I do see your point, but after a ton of conversation we decided to separate the install from upgrade (as you originally had it); then I decided to move the usage section come last since that was accurate for all audiences.

Reply:
ok, sounds good.